pkg/cmd/tests/tests_run.go

```diff
@@ -400,21 +423,22 @@ func (o *RunOptions) run(ctx context.Context, streams genericclioptions.IOStream
 		go func() {
 			defer wg.Done()
 
-			klog.V(2).InfoS("Waiting for process", "ID", e.id, "Process", e.cmd.String())
+			klog.V(2).InfoS("Waiting for process", "ID", e.id, "Command", e.cmd.String())
 			err := e.cmd.Wait()
 			if err != nil {
 				errs[entryIndex] = fmt.Errorf("can't wait for command %q with args %q: %w", e.cmd.Path, e.cmd.Args, err)
 			}
-			klog.V(2).InfoS("Process finished", "ID", e.id, "Process", e.cmd.String(), "ExitCode", e.cmd.ProcessState.ExitCode())
+			klog.V(2).InfoS("Process finished", "ID", e.id, "Command", e.cmd.String(), "ProcessState", e.cmd.ProcessState.String())
 		}()
 	}
+
+	wg.Wait()
+
 	err = apierrors.NewAggregate(errs)
 	if err != nil {
-		return err
+		klog.ErrorS(err, "Processes encountered errors")
```

Comment:
why is it no longer returning here?

Reply:
I figured that wasn't intentional to return here? Otherwise the runner never reaches logging the output of failed processes: https://github.com/rzetelskik/scylla-operator/blob/11db76fb389ed12d654133b061f4c7d00a9de50e/pkg/cmd/tests/tests_run.go#L433

Reply:
You can't ignore errors like `errors.New("exec: not started")` - those have to be manifested into our return code through errors. At the same time you are right about logs - this may be a good place to output them

Reply:
https://github.com/scylladb/scylla-operator/pull/1084